docs/pages/agents/agent-sdk.mdx

```diff
@@ -0,0 +1,121 @@
+# Agent SDK by XMTP
+
+This package provides the XMTP agent wrapper around [SDK for Node](https://github.com/xmtp/xmtp-js/tree/main/sdks/node-sdk).
+
+## Install
+
+:::code-group
+
+```bash [bun]
+bun install xmtp
+```
+
+```bash [yarn]
+yarn add xmtp
+```
+
+```bash [npx]
+npx install xmtp
+```
+
+:::
+
+## Overview
+
+These are the steps to initialize the AgentSDK listener and send messages.
+
+- `WALLET_PRIVATE_KEY`: The private key of the wallet that will be used to send or receive messages.
+
+```tsx
+import { XMTP } from "xmtp";
+
+const xmtp = new XMTP(onMessage, {
+  encryptionKey: WALLET_PRIVATE_KEY,
+});
+
+await xmtp.init();
+
+const onMessage = async (message, user) => {
+  console.log(`Decoded message: ${message.content.text} by ${user.address}`);
+
+  // Your AI model response
+  const response = "Hello from AI!";
+
+  //Send text message
```

Comment:
If this is a text message why does it have a reference to an `originalMessage`? I think original message only needs to be set for reactions and replies?